samples/core/volume_ops/volume_ops.py

```diff
@@ -20,11 +20,11 @@
     name="VolumeOp Basic",
     description="A Basic Example on VolumeOp Usage."
 )
-def volumeop_basic(size):
+def volumeop_basic(size='10M'):
     vop = dsl.VolumeOp(
         name="create_pvc",
         resource_name="my-pvc",
-        modes=dsl.VOLUME_MODE_RWM,
+        modes=dsl.VOLUME_MODE_RWO,
```

Comment:
I'd say revert the changes in this file.

Reply:
Any concerns of making this change? I want to make sure these samples are covered with the tests.

Reply:
I just saw that it is included in the e2e tests. Any particular reason of placing the tests in e2e instead of the sample test?

Reply:
Done.

Reply:
>Any particular reason of placing the tests in e2e instead of the sample test?

Not really. I couldn't get what's the difference so I randomly chose to place it there.
If it shouldn't be there, feel free to move it.

Reply:
This is good. Thanks for adding it.